x-pack/plugins/cloud_security_posture/server/create_indices/benchmark_score_mapping.ts

```diff
@@ -47,5 +47,8 @@ export const benchmarkScoreMapping: MappingTypeMapping = {
     low: {
       type: 'long',
     },
+    is_custom_score: {
```

Comment:
what about `is_unmuted_score` ? `custom` to me is very generic name, which might be good if we plan to add more 'signals' to the same score, but otherwise it's hard to undertand what custom refers to without knowing the history

Reply:
I think it's better than `custom` but how do you decide when to use `mute/unmute` and when `enable/disabled`. I think we should settle on one term and use it consistently, otherwise in the future it might get very confusing. I also thought that we decided to go with `mute/unmute` but I might be missing smth. Wouldn't block the pr on the naming ofc, but consider the renaming in the follow up if you think it makes sense